contributors/devel/OWNERS

```diff
@@ -0,0 +1,8 @@
+reviewers:
```

Comment:
maybe we focus on reviewers and turn on the bot without the approvers mechanism.  This way PRs are auto-assigned via blunderbuss, but we don't have a bottleneck.  Anyone can still lgtm like before.